src/Generators/Microsoft.Gen.AutoClient/Emitter.cs

```diff
@@ -489,4 +514,58 @@ private void EmitClassVariablesAndConstructor(RestApiType restApiType, string ht
         OutLn($"{optionsName} = autoClientOptions;");
         OutCloseBrace();
     }
+
+    private void EmitPathParameterValidation()
+    {
+        OutLn($$"""
+            private static bool IsPathParameterValid(string value)
+            {
+                if ({{String}}.IsNullOrWhiteSpace(value))
+                {
+                    return false;
+                }
+
+                #if NET6_0_OR_GREATER
+                if (value.Contains('/'))
```

Comment:
Is this the best we can do here? This is doing three full iteration of the string. Aren't there some overloads that take an array of chars to find so we only iterate once?

Reply:
IndexOfAny.